ios/FluentUI/SegmentedControl/SegmentPillButton.swift

```diff
@@ -35,9 +35,13 @@ class SegmentPillButton: UIButton {
         self.contentEdgeInsets = Constants.insets
 
         let title = item.title
-        self.setTitle(title, for: .normal)
         self.accessibilityLabel = title
         self.largeContentTitle = title
+        if let image = item.image {
+            self.setImage(image, for: .normal)
```

Comment:
do we need https://developer.apple.com/documentation/uikit/uilargecontentvieweritem/3183932-largecontentimage?language=objc ?

Reply:
Large content viewer works, shows the image and the title